newsroom/agenda/module.py

```diff
@@ -0,0 +1,16 @@
+from newsroom import MONGO_PREFIX
+from newsroom.agenda.service import FeaturedService
+from newsroom.agenda.model import FeaturedResourceModel
+
+from superdesk.core.module import Module
+from superdesk.core.resources import ResourceConfig, MongoResourceConfig
+
+
+featured_resource_config = ResourceConfig(
+    name="agenda_featured",
+    data_class=FeaturedResourceModel,
+    service=FeaturedService,
+    mongo=MongoResourceConfig(prefix=MONGO_PREFIX),
+)
+
+module = Module(name="newsroom.agenda_featured", resources=[featured_resource_config])
```

Comment:
As this module will be for normal Agenda items (Events & Planning items) and FeaturedAgenda items, I think the name of the module should just be `newsroom.agenda`